packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -8,6 +8,10 @@ import './DatePicker.scss';
 
 const propTypes = {
   /**
+   * Custom attributes to apply to the date input.
+   */
+  attrs: PropTypes.object,
```

Comment:
Agree. Renamed prop attrs to inputAttributes https://github.com/cerner/terra-core/pull/292/commits/ed6a858697a6cc5ee834047a74ad225eeca4db75